staging/src/kubevirt.io/client-go/kubecli/handler.go

```diff
@@ -10,6 +10,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 
 	virtv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/util"
```

Comment:
maybe alias this with `clientUtil` like in the other places?

Reply:
but in the other places, we had imported `util` pkg already, I think it's fine.

Reply:
and at the other places we are not in client-go, here we are, so it is clear which util package this is